man/systemd-syscfg.xml

```diff
@@ -0,0 +1,216 @@
+      </varlistentry>
+
+      <varlistentry>
+        <term><option>refresh</option></term>
+        <listitem><para>A combination of <option>unmerge</option> and <option>merge</option>: if already
+        mounted the existing <literal>overlayfs</literal> instance is unmounted temporarily, and then
+        replaced by a new version. This command is useful after installing/removing system configuration images,
+        in order to update the <literal>overlayfs</literal> file system accordingly. If no images are installed
+        when this command is executed, the equivalent of <option>unmerge</option> is executed, without establishing
+        any new <literal>overlayfs</literal> instance. Note that currently there's a brief moment where neither the
+        old nor the new <literal>overlayfs</literal> file system is mounted. This implies that all resources
+        supplied by a system extension will briefly disappear — even if it exists continuously during the
+        refresh operation.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><option>list</option></term>
+
+        <listitem><para>A brief list of installed configuraion images is shown.</para></listitem>
+      </varlistentry>
+
+      <xi:include href="standard-options.xml" xpointer="help" />
+      <xi:include href="standard-options.xml" xpointer="version" />
+    </variablelist>
+  </refsect1>
+
+  <refsect1>
+    <title>Options</title>
+
+    <variablelist>
+      <varlistentry>
+        <term><option>--root=</option></term>
+
+        <listitem><para>Operate relative to the specified root directory, i.e. establish the
+        <literal>overlayfs</literal> mount not on the top-level host <filename>/etc/</filename>
+        hierarchy, but below some specified root directory.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><option>--force</option></term>
+
+        <listitem><para>When merging system extensions into <filename>/etc/</filename>, ignore version
+        incompatibilities, i.e. force merging regardless of whether the version information included in
+        the configuration images matches the host or not.</para></listitem>
+      </varlistentry>
+
+      <xi:include href="standard-options.xml" xpointer="no-pager" />
+      <xi:include href="standard-options.xml" xpointer="no-legend" />
+      <xi:include href="standard-options.xml" xpointer="json" />
+    </variablelist>
+  </refsect1>
+
+  <refsect1>
+    <title>Exit status</title>
+
+    <para>On success, 0 is returned.</para>
+  </refsect1>
+
+  <refsect1>
+    <title>See Also</title>
+    <para>
+      <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
+      <citerefentry><refentrytitle>systemd-nspawn</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
+      <citerefentry><refentrytitle>systemd-sysext</refentrytitle><manvolnum>1</manvolnum></citerefentry>
+    </para>
+  </refsect1>
+
+</refentry>
```

Comment:
no new man page please, just reuse the sysext man page, and add an alias so that it covers both tools. 

See for example how the poweroff.xml doc handles all three: the "halt", the "reboot" and the "poweroff" command in one.

make sure to re-run "ninja -C update-man-rules" to regen the doc rules for meson after you set up the XML for the alias